packages/components/package.json

```diff
@@ -32,7 +32,7 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "build": "vite build && pnpm run --filter \"../onu-ui\" build",
+    "build": "vite build && vue-tsc --declaration --emitDeclarationOnly --skipLibCheck && pnpm run --filter \"../onu-ui\" build",
```

Comment:
I'm not sure what `vue-tsc` does, do we really need it?

Reply:
This is a solution to solve the problem that normal component type declaration files cannot be generated using `vite-plugin-dts`. Using `vite-plugin-dts`  causes the props type of some components to change to any.  But this is really a temporary plan, I will modify it.